cli/admin/admin.go

```diff
@@ -0,0 +1,92 @@
+package admin
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+
+	"github.com/apex/log"
+	"github.com/tarantool/cartridge-cli/cli/context"
+	"github.com/tarantool/cartridge-cli/cli/project"
+)
+
+const (
+	adminListFuncName = "__cartridge_admin_list"
```

Comment:
I suggest to introduce a new point to cli "style guide" - comment every struct field and every constant name.

Personally I guess how it will be used. But another person won't understand this code without detail reading.

Reply:
Added a comment.

Reply:
Where are comments for `adminHelpFuncName ` and `adminCallFuncName `?

Also I hope we should place a purpose of each function in such comments.

Reply:
Added one big comment with a link to cartridge-cli-extentions repo.